workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.IO;
+using System.Text;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class LaunchMenu
+    {
+        [MenuItem("SpatialOS/Mobile/Launch on Device", false)]
+        private static void LaunchMobileClient()
+        {
+            EditorUtility.DisplayProgressBar("Launching Mobile Client", "Installing APK", 0.3f);
```

Comment:
probably also need a check whether the user actually has adb installed on their machine